acceptance/acceptance_test.go

```diff
@@ -1764,6 +1764,45 @@ func testAcceptance(
 						})
 					})
 
```

Comment:
Couldn't figure out how to write the acceptance test. This is based off the cache-image test, but failing.

Reply:
Let's not bother with an acceptance test for this.

They're slower, require more resources, and are more likely to fail. We should reserve them for more critical code paths with more moving parts. Although this PR touches a few moving parts, we have plenty other acceptance tests that the reduce need for this one.

Reply:
@aemengo I did observe that it took a bit of time. I added the acceptance test because it was suggested to do so in the [issue](https://github.com/buildpacks/pack/issues/897#issuecomment-853059556). If @jromero says it's alright without this test then I agree that we can skip it.